README.md

```diff
@@ -101,6 +101,10 @@ Places the following files in the destination:
 
 * `version`: The version identified in the file name.
 
+* `version`: The version identified in the file name.
+
+* `skip_download`: *Optional.* Skip downloading object from S3. Same parameter as source configuration but used to define/override by get. Value need to be a true/false string.
```

Comment:
This should actually be moved to below under the `#### Parameters` section

Reply:
you are right, it have been moved